### ISSUE TITLE
Customize the Powershell prompt to show the name of the container

### ISSUE THREAD

**Issue by martonnagy** [microsoft/navcontainerhelper#1722]
**Describe the issue**
I have an idea: would it be possible to customize the powershell prompt in the container to show the name of the container? If you have a scenario when multiple prompt are open in a split view it is find to identity the correct window

**Screenshots**
Something like this:
![image](https://user-images.githubusercontent.com/63159395/108605660-738ba480-73ad-11eb-8b26-ce190052cf6f.png)

**Additional context**
I've changed the prompt.ps1 file by adding this function:
```
function global:prompt {
    Write-Host -Object "[Docker] containerName"
    return "> "
}
```

I am more than happy to contribute but I was unable to find the code what generates the prompt.ps1 file...if someone can point me to the right direction.

Cheers

**Comment by freddydk** [comment]
The prompt.ps1 is part of the generic image - it isn't generated.
location: https://github.com/microsoft/nav-docker/blob/master/generic/Run/Prompt.ps1

Currently enter-bccontainer in PowerShell ISE will actually show the server name, but I think it is a good idea to move that to prompt.ps1 - I have added it to the file above.

That will be included in the next update of generic images.
Unfortunately, currently, the prompt.ps1 cannot be overridden:-(

**Comment by freddydk** [comment]
I changed my mind and added the prompt to both the PowerShell and the command prompt shortcuts when creating these in New-BcContainer.

**Comment by martonnagy** [comment]
Hi Freddy, cool! Much appreciated!

**Comment by freddydk** [comment]
Thanks for the idea:-)

**Comment by freddydk** [comment]
Shipped in BcContainerHelper 2.0.6
